views/blog-index.jade

```diff
@@ -0,0 +1,182 @@
+extends layouts/main
+
+block content
+  .row
+    .large-12.columns
+      h1.main-page-title= pageTitle
+
+  .row
+    .large-12.columns
+      article.row.portfolio-item
+        .large-7.medium-5.small-12.columns
+          .image.th
+            img(src='/images/content/projects/scenes/full/onereview.jpg', alt='One Review', title='One Review')
+        .description.large-5.medium-7.small-12.columns
+          <a href="/blog/detail/part-1-interactive-javascript-map-of-canada-with-raphael">
+          h2.orange-text.orange-hover Post Title
+          </a>
+          p June 10 2016
+          p
```

Comment:
If you need more vertical whitespace add a css class and using margin instead of empty HTML elements. This keeps the markup clean.